internal/splitmerge/splitmerge_test.go

```diff
@@ -11,6 +12,28 @@ import (
 	"golang.org/x/sync/errgroup"
 )
 
+type BufferCloser struct {
+	bytes.Buffer
+	io.Closer
+	closed atomic.Bool
```

Comment:
We already have atomic  package in WAL-G, try to use it instead of importing the new one.

Reply:
Wow! There are more than one implementation of atomic boolean in Go!